editions/tw5.com/tiddlers/howtos/How to turn off Camel Case Linking.tid

```diff
@@ -0,0 +1,19 @@
+created: 20161209172820513
+modified: 20170101212038930
+tags: [[Customise TiddlyWiki]] Learning
+title: How to turn off Camel Case Linking
+type: text/vnd.tiddlywiki
+
+CamelCase is used in most Wikis, but it can interfere with text presentation in some situations.
+
+!! Turning off Camel Case locally
```

Comment:
make it CamelCase everywhere, in the tesxt and in the title, it's never two words, it's all in the name